feedback/listener.go

```diff
@@ -237,3 +241,20 @@ func WaitTimeout(wg *sync.WaitGroup, timeout time.Duration) bool {
 		return true // timed out
 	}
 }
+
+func (l *Listener) StartPprofServer() error {
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+
+	server := &http.Server{
+		Addr:    "127.0.0.1:8081",
```

Comment:
can't this cause conflicts with other ports set in the app? if so, I'd parameterize and read from env var

Reply:
I don't think this is necessary at all, since it's using statsd underneath.

Reply:
I've added the parameterization for the profiling http server.